config/global.ini.php

```diff
@@ -566,6 +566,11 @@
 ; Recommended to keep enabled for best security.
 auth_token_rotation_notification_days = 180
 
+; Number of days before the expiration date of a personal auth token, where an email notification is sent to the user.
+; If set to -1 days, notifications won't be sent. 
```

Comment:
```suggestion
; If set to 0 days, notifications won't be sent.
```

The code works the same with `0` and `-1`, having two really similar values documented differently feels a bit off.

And for the sake of "we actually don't want to send anything if the config values is 0 *or lower*", I would say we update both `getPeriodThreshold` functions (expiration + notification), to check for `$periodDays > 0`.

Reply:
updated the config to state that the feature is disabled when set to 0, and updated the logic for both notification providers to check if the value is greater than 0.